### PR TITLE
kernel plugin: vmlinuz -> kernel.img hard link

### PR DESCRIPTION
**Pull request by piso77**
Adhere to the new naming scheme decided in Heidelberg, and create a kernel.img copy of vmlinuz.

Signed-off-by: Paolo Pisati <paolo.pisati@canonical.com>

### CODE REVIEW

**Comment by sergiusens** [comment]
Can you update the unit tests for this case?

File: snapcraft/plugins/kernel.py [hunk 1]
```diff
@@ -311,6 +311,7 @@ def _copy_vmlinuz(self):
         os.link(src, dst)
         # why oh why?
         os.link(src, os.path.join(self.installdir, 'vmlinuz'))
```
Comment (sergiusens) [inline snapcraft/plugins/kernel.py]: do we still need this line?

Comment (kyrofa) [inline snapcraft/plugins/kernel.py]: And should they be link_or_copies?

Comment (sergiusens) [inline snapcraft/plugins/kernel.py]: that should come later if needed. This plugin is full of `os.link` calls ;-)

**Comment by piso77** [comment]
Ok, i removed the vmlinuz hard link, the comment and i've pushed again - once this get in, we might rewrite to use symlinks instead of hard links.

**Pull request merged**